packages/pds/src/services/record/index.ts

```diff
@@ -2,15 +2,21 @@ import { CID } from 'multiformats/cid'
 import { AtUri } from '@atproto/uri'
 import * as common from '@atproto/common'
 import { WriteOpAction } from '@atproto/repo'
+import {
+  atUriFormat,
+  didFormat,
+} from '@atproto/lexicon/src/validators/primitives'
```

Comment:
these verify fns will be in their respective packages after this PR:
https://github.com/bluesky-social/atproto/pull/576

no need to change tho